### ISSUE TITLE
Update Orion to Auro

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/OrionStatelessComponents__ods-inputoptions#10]
# User story

As a user, I want to be able to use the latest and greatest UI, so I can be awesome with Auro.

## Outline subtasks or tasks

> Decide which specific steps need to be completed and who is responsible for each of them.

- [x] remove Orion UI
- [x] remove the dependency on shape.json (optional)
- [ ] update UI of radio buttons and checkboxes to fit Auro UI spec
- [x] update `"name": "@alaskaairux/ods-inputoptions",` to `"name": "@alaskaairux/auro-inputoptions",`
- [ ] update Auro Doc site with new component 

Once the PR is merged and all work completed: 
- [ ] update repo name to be `AlaskaAirlines/auro-inputoptions`

## Definition of “Done” (exit criteria)

> The story is generally “done” when the user can complete the outlined task, but make sure to define what that is.

This story will be considered complete once the radio button and checkbox elements fit the new Auro design spec, and the next major release has a new npm name and the doc site is updated with the new content.

**Comment by tsalright** [comment]
Was mistakenly attached to the PR to complete the ODS refactor. Reopened so we can start the work on converting to Auro.

**Comment by gusnaughton** [comment]
@blackfalcon @tsalright Here are the list of tokens I derived from @vidalmenAS's mocks. 
To match some of these mocks, the border color had to be pulled from the text block. Will we need new tokens in the right category for these?

```
Checkbox:

    Normal:

        Text: 
            font-color: --auro-color-text-primary-on-light
        Unselected
            background: --auro-color-background-lightest
            border-radius: 6px
            border: 1px solid --auro-color-text-primary-on-light
            height: --auro-lg
        Checked
            background: --auro-color-ui-default-on-light
            border-radius: 6px
            border: 1px solid --auro-color-border-active-on-light
            height: --auro-lg
        Focus-Visible:
            border-radius: --auro-xxxs
            border: 2px solid --auro-color-ui-default-on-light
            height: 28px;
        Barred
            background: --auro-color-background-lightest
            border-radius: 6px
            border: 1px solid --auro-color-text-primary-on-light
            height: --auro-lg
    OnDark:
        Text: 
            font-color: --auro-color-text-primary-on-dark
        Unselected
            background: --auro-color-base-black-opacity-15
            border-radius: 6px
            border: 1px solid --auro-color-border-divider-on-dark
            height: --auro-lg
        Checked
            background: --auro-color-ui-default-on-dark
            border-radius: 6px
            border: 1px solid --auro-color-border-active-on-dark
            height: --auro-lg
        Focus-Visible:
            border-radius: --auro-xxxs
            border: 2px solid --auro-color-ui-default-on-dark
            height: 28px;
        Barred
            background: --auro-color-base-black-opacity-15
            border-radius: 6px
            border: 1px solid --auro-color-border-divider-on-dark
            height: --auro-lg

Spacing between Checkbox and Label: --auro-xs
```

**Comment by blackfalcon** [comment]
This comment has been moved to a new issue, https://github.com/AlaskaAirlines/AuroDesignTokens/issues/27

**Comment by blackfalcon** [comment]
This story is being re-scoped to the following: 

1. new repo created, auro-checkbox
1. move working code to new repo
1. apply Auro UI

Radio buttons has been de-scoped and will be done at a later date. New story needed. Option to have Radio buttons powers by Sass only option to be considered.

**Comment by blackfalcon** [comment]
Closing this PR as we are taking a new dev strategy.
